.github/workflows/main.yml

```diff
@@ -0,0 +1,123 @@
+name: CI
+
+# Controls when the workflow will run
+on:
+  # Triggers the workflow on push or pull request events but only for the main branch
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+jobs:
+  windows:
+    # The type of runner that the job will run on
+    runs-on: windows-latest
+
+    strategy:
+      matrix:
+        gmakeconfig:
+          - debug
+          #- profile
+          #- release
+        gmakeplatform:
+          - 64
+        ocamlversion:
+          - 4.13.x
+
+    steps:
+      - name: Checkout ocaml-orx
+        uses: actions/checkout@v2
+        with:
+          path: ocaml-orx
```

Comment:
The problem here is that by changing the configuration of the checkout action, the opam files are extracted outside the default search path.

Reply:
Ah of course - that makes perfect sense. Thank you for taking time to look at this!